code/integration-tests/runtime-tests/src/utils/subsquid/apollo/queries/bondedFinance.ts

```diff
@@ -0,0 +1,15 @@
+import { gql as gqlCore, gql as gqlReact } from "@apollo/client/core";
+
+const isFrontend = false;
+const gql = isFrontend ? gqlCore : gqlReact;
```

Comment:
I like this approach but we need to agree with FE on how to centralize the gql queries. Also, we can probably do this only once by exporting `gql` from some dedicated file that only imports both versions, and exports the right one, and then we import from this file every time and avoid repeating this logic.